android/aepsdk-bom/sdk-bom/build.gradle

```diff
@@ -32,26 +32,126 @@ configurations.all {
 
 dependencies {
     implementation 'androidx.appcompat:appcompat:1.6.1'
-    implementation 'com.adobe.marketing.mobile:core:latest.release'
-    implementation 'com.adobe.marketing.mobile:lifecycle:latest.release'
-    implementation 'com.adobe.marketing.mobile:identity:latest.release'
-    implementation 'com.adobe.marketing.mobile:signal:latest.release'
-    implementation 'com.adobe.marketing.mobile:userprofile:latest.release'
-    implementation 'com.adobe.marketing.mobile:assurance:latest.release'
-    implementation 'com.adobe.marketing.mobile:edge:latest.release'
-    implementation 'com.adobe.marketing.mobile:edgeidentity:latest.release'
-    implementation 'com.adobe.marketing.mobile:edgeconsent:latest.release'
-    implementation 'com.adobe.marketing.mobile:edgebridge:latest.release'
-    implementation 'com.adobe.marketing.mobile:edgemedia:latest.release'
-    implementation 'com.adobe.marketing.mobile:messaging:latest.release'
-    implementation 'com.adobe.marketing.mobile:optimize:latest.release'
-    implementation 'com.adobe.marketing.mobile:places:latest.release'
-    implementation 'com.adobe.marketing.mobile:analytics:latest.release'
-    implementation 'com.adobe.marketing.mobile:media:latest.release'
-    implementation 'com.adobe.marketing.mobile:target:latest.release'
-    implementation 'com.adobe.marketing.mobile:campaign:latest.release'
-    implementation 'com.adobe.marketing.mobile:campaignclassic:latest.release'
-    implementation 'com.adobe.marketing.mobile:audience:latest.release'
+    implementation('com.adobe.marketing.mobile:core') {
+        version {
+            strictly '[2.0.0, 3.0.0['
```

Comment:
nit: Can you extract this version string?

Reply:
https://docs.gradle.org/current/userguide/rich_versions.html I'd suggest starting to use Gradle's rich version. It will be easier to add some `reject` versions to skip the specific builds for extensions.